### PR TITLE
feat: add optioncategories to button category

### PR DESCRIPTION
**Pull request by BettyBrian**
Added optioncategories to the 'Button' category prefabs and removed advanced.ts where needed

### CODE REVIEW

File: src/prefabs/appBar.tsx [hunk 1]
```diff
@@ -1,15 +1,15 @@
 import { Icon, prefab, variable } from '@betty-blocks/component-sdk';
 import { AppBar } from './structures/Appbar';
 import { OpenPageButton } from './structures/OpenPage';
-import { options } from './structures/OpenPage/options';
+import { openPageButtonOptions } from './structures/OpenPage/options';
```
Comment (SanderVanRiessen) [inline src/prefabs/appBar.tsx:4]: import from structure

File: src/prefabs/cardActions.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { Icon, prefab, variable } from '@betty-blocks/component-sdk';
 import { CardActions } from './structures/CardActions';
 import { OpenPageButton } from './structures/OpenPage';
-import { options as openPageButtonOptions } from './structures/OpenPage/options/index';
+import { openPageButtonOptions } from './structures/OpenPage/options/index';
```
Comment (SanderVanRiessen) [inline src/prefabs/cardActions.tsx:4]: import from structure

File: src/prefabs/dialog.tsx [hunk 3]
```diff
@@ -20,7 +20,7 @@ import { options as rOptions } from './structures/Row/options';
 import { options as cOptions } from './structures/Column/options';
 import { options as bOptions } from './structures/Box/options';
 import { options as tOptions } from './structures/Text/options';
-import { options as buttonOptions } from './structures/Button/options';
+import { buttonOptions } from './structures/Button/options';
 import { Button } from './structures/Button';
```
Comment (SanderVanRiessen) [inline src/prefabs/dialog.tsx:24]: import from structure
this counts for 13 until 24

File: src/prefabs/logoutButton.tsx [hunk 4]
```diff
@@ -7,7 +7,7 @@ import {
   variable,
 } from '@betty-blocks/component-sdk';
 import { Button } from './structures/Button';
-import { options } from './structures/Button/options';
+import { buttonOptions } from './structures/Button/options';
```
Comment (SanderVanRiessen) [inline src/prefabs/logoutButton.tsx:10]: import from structure

File: src/prefabs/appBar.tsx (hunk 1 above)
Comment (BettyBrian) [reply src/prefabs/appBar.tsx:4]: Imported from structure

File: src/prefabs/card.tsx [hunk 5]
```diff
@@ -13,7 +13,7 @@ import { CardMedia } from './structures/CardMedia';
 import { Text } from './structures/Text';
 import { OpenPageButton } from './structures/OpenPage';
 import { options as textOptions } from './structures/Text/options/index';
-import { options as openPageButtonOptions } from './structures/OpenPage/options/index';
+import { openPageButtonOptions } from './structures/OpenPage/options/index';
```
[the remark this answers is not in the document]
Comment (BettyBrian) [reply src/prefabs/card.tsx:16]: Imported everything from the structure

File: src/prefabs/cardActions.tsx (hunk 2 above)
Comment (BettyBrian) [reply src/prefabs/cardActions.tsx:4]: Imported from the structure

File: src/prefabs/dialog.tsx (hunk 3 above)
Comment (BettyBrian) [reply src/prefabs/dialog.tsx:24]: Imported all the separate imports from ./structures now

**Review by sventruschel — APPROVED** [review]

**Pull request merged**